opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -228,6 +229,19 @@ message DoubleSummary {
   repeated DoubleSummaryDataPoint data_points = 1;
 }
 
+// DoubleGaugeHistogram represents the type of a double histogram metric that
+// always exports the "current value" for every data point. It should be used
+// for an "unknown" aggregation.
+//
+// A DoubleGaugeHistogram does not support different aggregation temporalities.
+// Given the aggregation is unknown, points cannot be combined using the same
+// aggregation, regardless of aggregation temporalities. Therefore,
+// AggregationTemporality is not included. Consequently, this also means
+// "StartTimeUnixNano" is ignored for all data points.
+message DoubleGaugeHistogram {
+  repeated DoubleHistogramDataPoint data_points = 1;
+}
+
```

Comment:
Description needs to be re-written. Aggregation is known, but temporality is unknown.

Reply:
@imccarten1 can you fix this?